src/roles/actors.rs

```diff
@@ -0,0 +1,356 @@
+#![cfg_attr(not(feature = "std"), no_std)]
+
+use rstd::prelude::*;
+use parity_codec::Codec;
+use parity_codec_derive::{Encode, Decode};
+use srml_support::{StorageMap, StorageValue, dispatch, decl_module, decl_storage, decl_event, ensure, Parameter};
+use srml_support::traits::{Currency, EnsureAccountLiquid};
+use runtime_primitives::traits::{Zero, Bounded, SimpleArithmetic, As, Member, MaybeSerializeDebug, MaybeDebug};
+use system::{self, ensure_signed};
+use crate::governance::{GovernanceCurrency, BalanceOf };
+
+use crate::traits::{Members, Roles};
+
+#[derive(Encode, Decode, Copy, Clone, Eq, PartialEq, Debug)]
+pub enum Role {
+    Storage,
+}
+
+
+#[cfg_attr(feature = "std", derive(Debug))]
+#[derive(Encode, Decode, Copy, Clone, Eq, PartialEq)]
+pub struct RoleParameters<T: Trait> {
+    // minium balance required to stake to enter a role
+    pub min_stake: BalanceOf<T>,
+
+    // the maximum number of spots available to fill for a role
+    pub max_actors: u32,
```

Comment:
Better to swap a position: first `min_actors`, then `max_actors`.
It's easier for perception like in counting: from smaller to bigger.